mate-base/mate-applets/mate-applets-1.28.0.ebuild

```diff
@@ -73,7 +73,6 @@ src_configure() {
 		$(use_with X x) \
 		$(use_with upower) \
 		$(use_with netlink nl) \
-		$(use_enable ipv6) \
```

Comment:
don' t you need to set `--enable-ipv6` here? Or was the flag removed upstream?

Reply:
On by default and doesn't actually do much any more disabling it so removed.